generators/add-web-assets/exc-react/templates/src/utils.js

```diff
@@ -32,10 +32,16 @@ async function actionWebInvoke (actionUrl, headers = {}, params = {}) {
   if (window.location.hostname === 'localhost') {
     actionHeaders['x-ow-extra-logging'] = 'on'
   }
+
+  // Switch between fetch method based on invocation params
+  const method = Object.keys(params).length ? 'post' : 'get'
```

Comment:
I don't think this would work for GET request with query params right?
There are scenarios for it, e.g. `https://mynamespace.adobeioruntime.net/api/v1/web/sample-app-0.0.1/get-customer?customerId=abc123`.
How about making `method` a separate param in the signature?

Reply:
@duynguyen I agree, 
But Ideally, developers don't have to know how the data is being sent. 
If they want to have more control they can edit the utils or extend it